Makefile

```
@@ -60,12 +60,15 @@ test: image-test
 # Deploy image to docker, assuming you are logged in with `docker login` already
 deploy-docker: image-prod
 	docker push sweavo/todo-app:prod
+	docker tag sweavo/todo-app:prod sweavo/todo-app:$$(git rev-parse HEAD)
+	docker push sweavo/todo-app:$$(git rev-parse HEAD)
```

_Very_ minor, but I believe we could push these together with:
```suggestion
	docker tag sweavo/todo-app:prod sweavo/todo-app:$$(git rev-parse HEAD)
        docker push sweavo/todo-app:prod --all-tags
```
[docs here](https://docs.docker.com/engine/reference/commandline/push/#options)